homeassistant/components/heos/quality_scale.yaml

```diff
@@ -34,15 +25,9 @@ rules:
   entity-unique-id: done
   has-entity-name: done
   runtime-data: done
-  test-before-configure: todo
+  test-before-configure: done
```

Comment:
This was errantly set to `todo`. The connection to the HEOS device is tested in the config flow.